compliance_checker/cf/cf.py

```diff
@@ -1893,14 +1894,12 @@ def check_calendar(self, ds):
                 continue
```

Comment:
nope. There's no requirement for a CF compliant dataset to only have one time coordinate.